esignet-service/cmd/esignet/main_test.go

```
@@ -31,6 +31,21 @@ func testHTTPClientConfig() config.HTTPClientConfig {
+		"/debug/pprof/",
+		"/debug/pprof/goroutine",
+		"/debug/pprof/heap",
+	}
+	for _, p := range paths {
+		rec := httptest.NewRecorder()
+		mux.ServeHTTP(rec, httptest.NewRequest(http.MethodGet, p, nil))
+		ts.Equal(http.StatusOK, rec.Code, "path %s", p)
+	}
+}
+
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)